integration-tests/chopsticks/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "e2e",
```

Comment:
This also deserves a better name, please 😃

Reply:
I am terrible in naming :smile:. Is e2e-tests better? : [48626b5](https://github.com/KILTprotocol/kilt-node/pull/614/commits/48626b53ebcdb7e1a8c66170da8c678e479e55b3)